Add support for onie_load command to account for loading the image from different flash types, for example serial NOR flash vs. parallel NAND flash

diff --git a/include/configs/onie_common_config.h b/include/configs/onie_common_config.h
index c941e2a..7fa17d4 100644
--- a/include/configs/onie_common_config.h
+++ b/include/configs/onie_common_config.h
@@ -37,15 +37,16 @@
 /*
  * onie_bootcmd load command, kicked off by CONFIG_BOOTCOMMAND.
  *
- * oniestart -- platform specific address of onie image in NOR flash
- * oniesz.b  -- size in bytes of the platform specific onie image in NOR flash
+ * oniestart -- platform specific address of onie image in flash
+ * oniesz.b  -- size in bytes of the platform specific onie image in flash
  */
 #define CONFIG_ONIE_BOOT_CMD \
 	"nos_bootcmd=echo\0"						\
+	"onie_load=cp.b $onie_start $loadaddr ${onie_sz.b}\0"\
 	"onie_bootcmd=echo Loading Open Network Install Environment ...; " \
 	  "echo Platform: $onie_platform ; "				\
 	  "echo Version : $onie_version ; "				\
-	  "cp.b $onie_start $loadaddr ${onie_sz.b} && "			\
+	  "run onie_load && "			\
 	  "run onie_args && bootm ${loadaddr}\0"
 
 /*
